components/event_dispatcher.rst

```diff
@@ -135,7 +135,7 @@ A call to the dispatcher's ``addListener()`` method associates any valid
 PHP callable to an event::
 
     $listener = new AcmeListener();
-    $dispatcher->addListener('acme.action', array($listener, 'onFooAction'));
+    $dispatcher->addListener('foo.action', array($listener, 'onFooAction'));
```

Comment:
Due to naming convention described in the previous line of the doc I don't think that naming the event `'acme.foo.action'` is the better choice.
Maybe using `'acme_foo.action'` can be a prefered approach in regard of these naming convention :
* __"Prefix names with a namespace followed by a dot"__
* __"End names with a verb"__